.github/actions/set-username-and-hostname/action.yml

```diff
@@ -4,6 +4,7 @@ inputs:
   user:
     description: the username that will be used for following steps
     required: true
+    default: ${{github.triggering_actor}}
```

Comment:
required and default?

Reply:
lol yep. shrugs.
It's mainly there because I found it too confusing to find the default over in the other file.